invenio_app_ils/documents/mappings/os-v1/documents/document-v1.0.0.json

```diff
@@ -617,7 +645,9 @@
                 "type": "keyword"
               },
               "title": {
-                "type": "text"
+                "type": "text",
```

Comment:
Check with all on what fields need accent removing, and update accordingly based on review

Reply:
this will be quite intense change, if we start considering all of the fields. I would leave it at the title for the moment, and seek feedback if it helped.

Reply:
Just the main title or the other title fields too (as it is currently)?